src/cbapi/psc/livequery/models/device_summary.yaml

```diff
@@ -0,0 +1,37 @@
+type: object
+required: [] # TODO(ww): Find out which result fields are required
+properties:
+  id:
+    type: string
+    description: The result's unique ID
+  total_results:
+    type: integer
+    format: int32 # NOTE(ww): This is a guess
+    description: Number of results returned for this particular device
+  device_id:
+    type: integer
+    format: int32 # NOTE(ww): This is a guess
+    description: The device's unique ID
+  device_name:
+    type: string
+    description: The device's name
+  time_received:
+    type: string
+    description: The time at which this result was received # NOTE(ww): This is a guess
+  status:
+    type: string
+    description: The result's status
+  device_message:
+    type: string
+    description: Placeholder # TODO(ww): Needs description
+  policy_id:
+    type: integer
+    format: int32 # NOTE(ww): This is a guess
+    description: The device's policy ID
+  policy_name:
+    type: string
+    description: The device's policy name
+  metrics: # TODO(ww): Document each field in metrics?
```

Comment:
Metrics is not an object but an array of objects we might want to make this more clear
```
"metrics": [
      {
           "key": "",
           "value": 0.0
      },
]
```

Reply:
I copied that from result.yaml.  Maybe that should be changed, too? Or are these different things? In any event, I'm changing this one to "array of object."

Reply:
I referred to our developer.carbonblack.com website for the documented format

Reply:
It's now been changed to "array of object." Should I extend that to actually put the "key" and "value" fields in under the subobject?

Reply:
It would match your other model you created which has the metrics property and it gives perspective what the items of the array are

Reply:
Added.